src/ioctls/vm.rs

```diff
@@ -610,6 +610,62 @@ impl VmFd {
         Ok(())
     }
 
+    /// Enable the specified capability as per the `KVM_ENABLE_CAP` ioctl.
+    ///
+    /// See the documentation for `KVM_ENABLE_CAP`.
+    ///
+    /// Returns an io::Error when the capability could not be enabled.
+    ///
+    /// # Arguments
+    ///
+    /// * kvm_enable_cap - KVM capability structure. For details check the `kvm_enable_cap`
+    ///                    structure in the
+    ///                    [KVM API doc](https://www.kernel.org/doc/Documentation/virtual/kvm/api.txt).
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// # extern crate kvm_ioctls;
```

Comment:
Please remove his line (and the next one prefixed by `#`). They don't show up in the generated doc and the doctest passes without them.

Reply:
Fixed but those lines (prefixed by #) where there in every other doctest.

Reply:
I can't remove it, the CI complains:
```

---- src/ioctls/vm.rs - ioctls::vm::VmFd::enable_cap (line 627) stdout ----
--

```